content/modules/ROOT/pages/rhads-ai/rhads-ai-rhads/lab-ai-rhads-overview-rhads-ai.adoc

```diff
@@ -6,34 +6,27 @@ These labs will guide you through building and deploying a secure AI agent using
 
 == Objectives
 
-This lab demonstrates how to use a pre-built software template in RHADS to deploy an AI Agent to OpenShift. This approach helps organizations ensure that developers have the tools and security standards they need right from the beginning. We will use the software templates already integrated into the Red Hat Developer Hub.
+This lab demonstrates how to use a pre-built software template in RHADS to deploy an AI Agent to OpenShift. This approach helps organizations ensure that developers have the tools and security standards they need right from the beginning. We will use the software templates already integrated into {product_rhdh_name}.
 
-This guide is divided into two labs, each focusing on a different role:
+This guide is divided into two labs, each focusing on a different role; AI Platform Engineer and AI Engineer.
 
 == AI Platform Engineer
 
-As a member of the AI Platform Engineering team, you are responsible for providing the tools, templates, and security guidelines for the AI engineering team to build applications in OpenShift.
+As a member of the AI Platform Engineering team, you are responsible for providing the tools, templates, and security guidelines for the AI engineering team to build applications on OpenShift.
```

Comment:
I reviewed this with AI. the answer is 

You should use "in OpenShift."

Your first version was correct.

In OpenShift (Correct): This is the standard idiom. You build applications within a platform or environment.

On OpenShift (Less common): While people might understand you, "on" usually refers to applications running on a platform. For the act of building or developing, "in" is the better choice.